index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="IE=edge">
+    <title>Lightbeam</title>
+    <meta name="description" content="Lightbeam is a Firefox add-on that uses interactive visualizations to show you the relationships between third parties and the sites you visit.">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- <link rel="stylesheet" href="style.css"> -->
+    <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
+    <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
+  </head>
+  <body>
+  <h1>Lightbeam</h1>
+  <!-- <script src="script.js"></script> -->
```

Comment:
As we will likely need this it is fine, code comments are pretty bad. (don't follow my example when I do it :D)